### DISCUSSION TITLE
[Question]: Google Search Grounding

### DISCUSSION THREAD

**Discussion by omarchouikha-goreply** [danny-avila/LibreChat#4200] [Q&A]
### What is your question?

Is there a way to add VertexAI Grounding with chat history?

### More Details

I have tried to implement it using the "@google-cloud/vertexai" module and I still can't get it to work correctly.
Can anyone please help out?

Thank You!

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by insilications** [comment]
It seems that there are some changes to be made to LibreChat's UI in order to implement this... https://ai.google.dev/gemini-api/docs/grounding https://ai.google.dev/gemini-api/docs/grounding/search-suggestions

**Reply by ChristianPilot** [reply]
Is there a way to toggle this default true? :)
